signerv2/remote_signer_client.go

```diff
@@ -0,0 +1,107 @@
+package signerv2
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/Layr-Labs/eigensdk-go/utils"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+type RpcRequest struct {
+	JsonRPC string      `json:"jsonrpc"`
+	Method  string      `json:"method"`
+	Params  interface{} `json:"params"`
+	ID      int         `json:"id"`
+}
+
+type RemoteSignerClient interface {
+	SignTransaction(from common.Address, tx *types.Transaction) (*types.Transaction, error)
+}
+
+// RemoteSigner is a client for a remote signer
+// It implements Consensys Web3 Signer
+// Reference: https://docs.web3signer.consensys.io/reference/api/json-rpc
```

Comment:
you say that we implement the web3 signer jsonrpc interface, but the interface above only supports 1 of the 5 supported methods: https://docs.web3signer.consensys.io/reference/api/json-rpc#eth_signtransaction

Reply:
yea because we don't need others now. Maybe I should just says it implements just `eth_signtransaction` for now. good call

Reply:
7a6df19465ed37185f3719c841bd4ec3d2e9ddaf